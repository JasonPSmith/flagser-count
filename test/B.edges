0 1
1 2
2 0
3 0
3 1
3 2
4 0
4 1
4 2
