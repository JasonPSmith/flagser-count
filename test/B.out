# [euler_characteristic cell_count_dim_0 cell_count_dim_1 ...]
2 5 9 6
